### PR TITLE
Ck 37 create a user account back end

### PR DESCRIPTION
**Pull request by Tosvng**

### CODE REVIEW

**Review by Amikz — APPROVED** [review]
LGTM, thanks!

**Review by Ahsen19 — APPROVED** [review]

File: .github/workflow/unitTest.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: Unit tests
+
+on:
+  pull_request:
+    branches: [deploy]
+  push:
+    branches: [deploy]
```
Comment (JoshuaDueck) [inline .github/workflow/unitTest.yml:7]: We don't have a deploy branch, is this something new?

File: server/.env [hunk 2]
```diff
@@ -1,2 +1,3 @@
 PORT=4000
-MONGO_URL="mongodb+srv://chorekeeper:chorekeeper@chorekeeper.lukufhv.mongodb.net/?retryWrites=true&w=majority"
\ No newline at end of file
+MONGO_URL="mongodb+srv://chorekeeper:chorekeeper@chorekeeper.lukufhv.mongodb.net/?retryWrites=true&w=majority"
+JWT_SECRET="atokenthatnoonecanguessbecauseitisveryrandom"
```
Comment (JoshuaDueck) [inline server/.env:3]: Given that this is a public repo, I am concerned about this haha

Comment (Tosvng) [reply server/.env:3]: Yeah me too, I decided to leave it as TDD for now just to see how things work. I still looking into ways to pass those env variable securely.

File: .github/workflow/unitTest.yml (hunk 1 above)
Comment (Tosvng) [reply .github/workflow/unitTest.yml:7]: Lol my bad that's meant to be develop

File: server/.env (hunk 2 above)
Comment (JoshuaDueck) [reply server/.env:3]: I think GitHub has a repo secrets feature, but I have not used that before. Other than that, whatever environment we deploy on should have a place for things like that I'm sure... I don't know for sure though.

Comment (Tosvng) [reply server/.env:3]: I'll look into github secrets

**Review by JoshuaDueck — APPROVED** [review]

**Comment by JoshuaDueck** [comment]
There were merge conflicts because you deleted `hello.js`. I just resolved that, so you can merge whenever. We'll need to make some changes to the front-end now.

**Pull request merged**